deepchecks/tabular/checks/data_integrity/is_single_value.py

```diff
@@ -60,14 +66,17 @@ def run_logic(self, context: Context, dataset_type: str = 'train') -> CheckResul
 
         df = select_from_dataframe(df, self.columns, self.ignore_columns)
 
-        num_unique_per_col = df.nunique(dropna=False)
+        num_unique_per_col = df.nunique(dropna=self.ignore_nan)
         is_single_unique_value = (num_unique_per_col == 1)
 
         if is_single_unique_value.any():
             # get names of columns with one unique value
             # pylint: disable=unsubscriptable-object
             cols_with_single = is_single_unique_value[is_single_unique_value].index.to_list()
-            uniques = df.loc[:, cols_with_single].head(1)
+            uniques = pd.DataFrame({
+                column_name: [column.sort_values(kind='mergesort').values[0]]
```

Comment:
isn't sort redundant?
```suggestion
                column_name: [column.dropna().values[0]]
```

Reply:
note: 
I revert the change 
it will fail if to set `ignore_nan=False` and pass a data frame with only nans